Networking/Networking/CookieNonce/CookieNonceAuthenticator.swift

```diff
@@ -48,12 +47,12 @@ final class CookieNonceAuthenticator: RequestRetrier & RequestAdapter {
             // Only retry because of failed authorization
             case .responseValidationFailed(reason: .unacceptableStatusCode(code: 401)) = error as? AFError
         else {
-            return completion(false, 0.0)
+            return completion(.doNotRetry)
         }
 
         requestsToRetry.append(completion)
         if !isAuthenticating {
-            startLoginSequence(manager: manager)
+            startLoginSequence(manager: session)
```

Comment:
It seems like we should rename these parameters to `session` instead of `manager` too.

Reply:
👍 Addressed in https://github.com/woocommerce/woocommerce-ios/pull/12125/commits/a08f02809dfbfced14975b649500f023f2e49526.